### PR TITLE
Ft/dashboard layout

### PR DESCRIPTION
**Pull request by Shegsdev**
- Add Layout for dashboard

### CODE REVIEW

File: frontend/mms-Admin/components/Layout/SideBar/SideBar.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+}
+.active {
+  font-weight: 600 !important;
+}
+.sidebar-menu {
+    height: 80vh;
+    padding-top: 30px;
+    background: none !important;
+    border: none !important;
+    /* margin: 0 !important; */
```
Comment (emmsdan) [inline frontend/mms-Admin/components/Layout/SideBar/SideBar.css:16]: no unused code

File: frontend/mms-Admin/package-lock.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "app",
   "version": "0.1.0",
```
Comment (emmsdan) [inline frontend/mms-Admin/package-lock.json:3]: Please, revert package-lock back. upgrade your note to use Node 16+ (18 recommended)
do not have a package lock change when they is no package.json changes

**Review by emmsdan — CHANGES_REQUESTED** [review]
Please, attend to comment.

File: frontend/mms-Admin/package-lock.json (hunk 2 above)
Comment (Shegsdev) [reply frontend/mms-Admin/package-lock.json:3]: I didn't make any changes tho
I'll resolve it.

Comment (Shegsdev) [reply frontend/mms-Admin/package-lock.json:3]: There's been a lot of changes since we reconfigured the project setup

Comment (Shegsdev) [reply frontend/mms-Admin/package-lock.json]: I'll reverted the package-lock file but we still need to update it later

**Comment by dejetem** [comment]
it's okay

**Review by dejetem — APPROVED** [review]

**Pull request merged**